client/settings/state/reducer.js

```diff
@@ -1,14 +1,15 @@
 import { SET_FORM_PROPERTY } from './actions';
 import values from './values/reducer';
 import * as formValueActions from './values/actions';
+import mapValues from 'lodash/mapValues';
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/809/commits/ca0d444ba7d11b22d6a9a504355ba6dad2e396d9.